doc/eredis_cluster.md

```diff
@@ -0,0 +1,466 @@
+
```

Comment:
a bit weird that there is no comment that this file shouldn't be changed manually..but I guess thats how edoc is

Reply:
EDoc normally generates HTML. https://github.com/uwiger/edown is used for generating Markdown and it's not perfect.